scene/animation/animation_blend_tree.h

```diff
@@ -148,7 +148,7 @@ VARIANT_ENUM_CAST(AnimationNodeOneShot::MixMode)
 class AnimationNodeAdd2 : public AnimationNode {
 	GDCLASS(AnimationNodeAdd2, AnimationNode);
 
-	StringName add_amount = PNAME("add_amount");
```

Comment:
It's confusing enough having different names for "add_amount" and "blend_amount" when simply "amount" or "value" would do the job nicely.

I don't see a reason to change this or break compatibility, unless there's actually a reason why add2 and add3 need different parameter names.

Reply:
I put different names because add2 and add3 although both are float types they operate at different ranges and that way it's weird to merge them together, the slider would be inconsistent for one of them. The simplest solution I found was to make the names unique.

example without merge:

https://user-images.githubusercontent.com/1387165/177231332-37d06f48-b1c1-449c-b20c-9b7a86b58587.mp4